x-pack/plugins/ml/public/embeddables/anomaly_swimlane/anomaly_swimlane_initializer.tsx

```diff
@@ -76,30 +124,49 @@ export const AnomalySwimlaneInitializer: FC<AnomalySwimlaneInitializerProps> = (
 
   const isPanelTitleValid = panelTitle.length > 0;
 
+  const jobIdsErrors = getJobSelectionErrors(jobIds);
+
   const isFormValid =
     isPanelTitleValid &&
+    !jobIdsErrors &&
     (swimlaneType === SWIMLANE_TYPE.OVERALL ||
       (swimlaneType === SWIMLANE_TYPE.VIEW_BY && !!viewBySwimlaneFieldName));
 
   const resultInput = {
+    jobIds,
     panelTitle,
     swimlaneType,
     ...(viewBySwimlaneFieldName ? { viewBy: viewBySwimlaneFieldName } : {}),
   };
 
+  const newJobUrl = useMlLink({ page: ML_PAGES.ANOMALY_DETECTION_CREATE_JOB });
```

Comment:
Not necessarily for this PR but I noticed pretty much all the places using the `JobSelectorControl` are using the same job creation link - maybe it could be built into the component itself?

Reply:
Not all of the usage require the create job link, so I kept it at the parent component level. Different services in  `KibanaReactContextValue` make it a bit more difficult to implement, because in the context for alerting flyout does not contain the locator service